contracts/lease/src/contract/mod.rs

```diff
@@ -26,7 +26,8 @@ mod cmd;
 pub mod msg;
 mod state;
 
-const CONTRACT_VERSION: Version = 0;
+const EXPECTED_MIGRATION_STORAGE_VERSION: Version = 0;
```

Comment:
that should be an edge case because there is no version in the storage at all. The current implementation of `versioning::upgrade_contract` does not handle it and would fail.

Reply:
so if `CONTRACT_STORAGE_VERSION == ZERO` then the function should only store it

Reply:
Added a compatibility method to handle those cases. As marked with TODO, should be removed after all contracts have been migrated.